src/test/java/com/study/racing/step1/SetCollectionTest.java

```diff
@@ -0,0 +1,61 @@
+package com.study.racing.step1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.apache.logging.log4j.util.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+
+    @Test
+    @DisplayName("요구사항 1")
+    public void test1() {
```

Comment:
사용자가 테스트 코드로 들어왔을때 어떤 기능의 테스트인지
한눈에 보면 좋을 것 같아요!

```suggestion
@Test
@DisplayName("문자 크기 확인하기)
public void test1() {
```

```suggestion
@Test
public void 문자크기_확인하기() {
```